src/main/kotlin/org/team5419/frc2019offseason/subsystems/Vacuum.kt

```diff
@@ -62,13 +64,17 @@ class Vacuum(
 
     public override fun update() {
         // println("${mTalon.getOutputCurrent()}")
-        // hasPeice = hasPeice || mTalon.getOutputCurrent() >= Constants.Vacuum.RESTING_THRESHOLD
-        // if (hasPeice && !isPumping){
+        // hasPiece = hasPiece || mTalon.getOutputCurrent() >= Constants.Vacuum.RESTING_THRESHOLD
+        // if (hasPiece && !isPumping){
         //     mTalon.set(ControlMode.PercentOutput, 0.2)
         // }
-        // else if(!hasPeice && !isPumping){
+        // else if(!hasPiece && !isPumping){
         //     mTalon.set(ControlMode.PercentOutput, 0.0)
         // }
+        if (!hasPiece && mTalon.getOutputCurrent() >= Constants.Vacuum.RESTING_THRESHOLD) {
+            hasPiece = true
+            mVision.flashNumTimes(3)
+        }

```

Comment:
What happens if the hatch panel or ball gets dropped? How will `hasPiece` get set to `false`?